privacy_evaluator/output/user_output.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+
+
+class UserOutput:
+
+    """User Output Class"""
+
+    def __init__(
+        self,
+        attack_data_y: np.ndarray,
+        privacy_risk: np.ndarray,
+        all_labels: np.ndarray,
+    ):
+        """
+        Initilaizes the Class with values
+        """
+        self.attack_data_y = attack_data_y
+        self.privacy_risk = privacy_risk
+        self.all_labels = all_labels
+
+    def histogram_top_k(self, k: int = 10, show_diagram: bool = True):
+        """
+        Draws a historgam of the top k points with biggest Privacy score
+        :param k: the number of points to consider, default 10
+        :param show_diagram: determines if the diagram should be shown, default: True
+        :return: All lables of the data with the number of points that are in the top k
+        :rtype: tuple
+        """
```

Comment:
Maybe move this return type to the method declaration (e.g. `histogram_top_k(self, k: int = 10, show_diagram: bool = True) -> Tupel[np.ndarray, np.ndarray] :`)

Reply:
fixed it

Reply:
You can then just remove the `:type: ...` line in the docstring.